core/src/main/scala/org/ensime/util/Reporter.scala

```diff
@@ -2,13 +2,12 @@
 // License: http://www.gnu.org/licenses/gpl-3.0.en.html
 package org.ensime.util
 
-import org.slf4j.LoggerFactory
-
 import scala.reflect.internal.util.Position
 import scala.tools.nsc.reporters.Reporter
 
 import org.ensime.api._
 import org.ensime.core.PositionBackCompat
+import org.slf4j.LoggerFactory
```

Comment:
best not change files you're not otherwise touching (on larger projects these kinds of changes can be a real nightmare when you have 50 open PRs with conflicts across 10 team branches, because somebody's IDE "optimised" some imports in a file they weren't even changing)